include/boost/math/differentiation/autodiff.hpp

```diff
@@ -91,7 +115,7 @@ class fvar
   fvar() = default;
 
   // Initialize a variable or constant.
-  fvar(const root_type&, const bool is_variable);
```

Comment:
In this case, there was no warning, just a recommendation from Clang-Tidy (or ReSharper -- which one escapes me). Whatever the case, I think this should be reverted since the user should be able to look at that function (or any function where a `const` has similarly been removed from the function declaration, but is present in the function definition) and know that the variable passed in won't be modified. I'll go through and re-add the `const`s where they were removed in such cases.

Reply:
`const`s have been replaced